Rakefile

```diff
@@ -0,0 +1,6 @@
+# Add your own tasks in files placed in lib/tasks ending in .rake,
+# for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
+
+require_relative 'config/application'
+
+Rails.application.load_tasks
```

Comment:
I think we might like an explicit default rake task.

Would we want something like the following to make rubocop avail as a rake task?

```ruby
require 'rubocop/rake_task'
RuboCop::RakeTask.new
```

Would we want the default task to run spec, then rubocop?

```
task default: [:spec, :rubocop]
```

I believe for preservation_catalog, we had to write one no-op test so travis would be happy from the get go.  (That's probably how the controller confusion started)

Will we need to have travis do any postgres setup, or is this premature?

```
task :travis_setup_postgres do
  sh("psql -U postgres -f db/scripts/tcr_test_setup.sql")
end
```

Reply:
Created default Rake task, and created #5